src/__tests__/FamilyIntake.test.js

```diff
@@ -0,0 +1,37 @@
+import FamilyIntake from '../components/common/FamilyIntake';
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+// mocking matchmedia to bypass error.
```

Comment:
I'm wondering what the error was you were getting. Bypassing may be a good patch, but is the error actually 'resolved' please comment below :)

Reply:
![Screen Shot 2022-01-24 at 2 10 46 AM](https://user-images.githubusercontent.com/70237021/150738074-6e0a39a2-ff8c-40d2-aaa8-3d537276a1c4.png)

I've attached the error I️'m getting when attempting to run the test suite. I️ did a bit of research and found [this post](https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom) in the jest documentation. The first line states this error occurs "If some code uses a method which JSDOM (the DOM implementation used by Jest) hasn't implemented yet". In the same post, the code I used to mock matchMedia is provided. If there's something I️ need to change in my code, I️ will definitely take other recommendations.